squarelet/templates/organizations/organization_list.html

```diff
@@ -1,50 +1,96 @@
+    <nav class="return">
+      <a href="{% url "users:detail" username=user.username %}">
+        {% include "core/icons/arrow-left.svg" %}
+        {% trans "Back to your account" %}
       </a>
-    </div>
-  </div>
-  
-  <div class="_cls-expandedContent">
-    <div id="_id-autocomplete" class="_cls-autocomplete">
-      <div class="_cls-inputSection">
-        <div class="_cls-input">
-          <div class="_cls-searchCircle"></div>
-          <div class="_cls-searchRectangle"></div>
-          <div id="_id-close" class="_cls-closeButton">
-            <div class="_cls-closeOpaque"></div>
-            <div class="_cls-closeLeft"></div>
-            <div class="_cls-closeRight"></div>
+    </nav>
+    {% endif %}
+    <h1>{% trans "Join an organization" %}</h1>
+  </header>
+
+  <div class="content">
+    <section class="search">
+      <h2>{% trans "Find an organization to join" %}</h2>
+      <div id="org_search"></div>
+    </section>
+
+    <section class="invites">
+      <h2>
+        {% trans "Pending invitations" %}
+      </h2>
+      {% if not has_pending %}
+      <div class="empty">
+        <p>{% trans "Teams you can join will appear here when available." %}</p>
+      </div>
+      {% endif %}
+
+      {% if potential_orgs %}
+      <div class="pre-approved">
+        <header>
+          <h2>{% trans "Pre-approved" %}</h2>
+          <p>{% trans "Based on your email address, you're pre-approved to join:" %}</p>
+        </header>
+        <div class="orgs">
+          {% for organization in potential_orgs %}
+          <div class="invite">
+            {% include "account/team_list_item.html" %}
+            
+            <form method="post" action="{% url "organizations:detail" slug=organization.slug %}" class="actions join">
+              {% csrf_token %}
+              <button class="button primary" name="action" value="join">
+                <img src="{% static "icons/check.svg" %}">
+                {% trans "Join" %}
+              </button>
+            </form>
           </div>
-          <input id="_id-filter" placeholder="Filter..." />
-          <div class="_cls-bottomLine"></div>
+          {% empty %}
+          <p>{% trans "No pre-approved teams are available to join." %}</p>
+          {% endfor %}
         </div>
-        <div class="_cls-inputPad"></div>
       </div>
-      <div class="_cls-results">
-
-        <div class="_cls-resultsList" id="_id-results">
-          {% for org in organization_list %}
-            <a href="{% url 'organizations:detail' org.slug %}">
-              <div class="_cls-result">
-                <div class="_cls-inlineAvatar">
-                  {% avatar org 22 %}
-                </div>
-                <span class="_cls-inlineText">
-                  {{ org.name }}
-                </span>
-              </div>
-            </a>
-          {% endfor %}
+      {% endif %}
+
+    {% if invitations %}
+    <div class="pending">
+      <h2>{% trans "Pending requests" %}</h2>
```

Comment:
This isn't distinguishing between invitations and requests right now, but each variant of the "invitation" model has distinct actions and relationship to the user.

1. An invitation is sent to the user, and has actions like "Accept" or "Reject"
2. A request is sent from the user, and has a "Withdraw" action.

We should probably be passing two different data through the context for these, and rendering them as distinct sections with distinct actions.